src/main/resources/com/google/api/codegen/java/main.snip

```diff
@@ -452,9 +452,13 @@
       {@pathTemplateCheckLine(pathTemplateCheck)}
     @end
     {@apiMethod.serviceRequestTypeName} request =
-        {@apiMethod.serviceRequestTypeName}.newBuilder()
-        {@builderSetCalls(apiMethod.requestObjectParams)}
-        .build();
+        @if apiMethod.requestObjectParams
+          {@apiMethod.serviceRequestTypeName}.newBuilder()
+              {@builderSetCalls(apiMethod.requestObjectParams)}
```

Comment:
Can `@builderSetCalls` method return nothing if `apiMethod.requestObjectParams` is empty? I is subjective, but it seems like moving the check to the `@builderSetCalls` snip method is cleaner, than having the if-else statement here. Also it can reduce code duplication (seems same check is done on L613)

Reply:
I did this because even if `@builderSetCalls` returns nothing, there will be an empty newline in the caller method. This is some uncommon settings about snippet that I never understand... That's also why I wrapped `@builderSetCalls` in an if-else statement.